js/admin.js

```diff
@@ -110,6 +110,7 @@ jQuery( document ).ready(function($) {
 							}
 						}
 					};
+					$('.progress-bar').addClass('install-error');
```

Comment:
I agree with @grappler . We should change background color as well as message in the bar.